mesos-docker/run/run.sh

```diff
@@ -354,7 +355,7 @@ function start_slaves {
 
   for i in `seq 1 $NUMBER_OF_SLAVES`;
   do
-    start_slave_command="nohup /usr/local/bin/wrapdocker ; /usr/sbin/mesos-slave --master=$master --ip=$dip $(quote_if_non_empty $MESOS_SLAVE_CONFIG)"
+    start_slave_command="nohup /usr/local/bin/wrapdocker; /usr/sbin/mesos-slave --master=$master --work_dir=/tmp --ip=$dip $(quote_if_non_empty $MESOS_SLAVE_CONFIG)"
```

Comment:
Does Mesos 1.0 still call the program `mesos-slave` or have they moved to the more neutral term, `mesos-agent` (or whatever)?